.github/workflows/server-build.yml

```
@@ -175,6 +175,11 @@ jobs:
           APPSMITH_ENVFILE_PATH: /tmp/dummy.env
           APPSMITH_VERBOSE_LOGGING_ENABLED: false
         run: |
+          if [[ "${{ inputs.is-pg-build }}" == "true" ]]; then
+            export APPSMITH_DB_URL="jdbc:postgresql://localhost:5432/postgres"
```

We'll need to set the username/password here now no? Something like:

```suggestion
            export APPSMITH_DB_URL="jdbc:postgresql://postgres:password@localhost:5432/postgres"
```

---

@sharat87 it's failing for the above connection string in my local, but adding a separate username and password works. Let me add 
```
Connection Fail:
spring.datasource.url=jdbc:postgresql://postgres:password@localhost:5432/postgres

Connection successful:
spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
spring.datasource.username=postgres
spring.datasource.password=password
```

From little search I think `jdbc:postgresql://hostname:port/database` is the valid format. I think we need to add more env variable to fix this issue.

---

Created another PR pointing to PG branch to address this issue https://github.com/appsmithorg/appsmith/pull/33543

---

@sharat87 extracting username and password worked in my local please check app/server/appsmith-server/src/main/java/com/appsmith/server/configurations/CommonDBConfig.java, we can live with single env APPSMITH_DB_URL for both Mongo and Postgres.